app/src/main/java/openfoodfacts/github/scrachx/openfood/fragments/AllergensAlertFragment.java

```diff
@@ -92,6 +100,8 @@ public void onViewCreated(@NonNull final View view, @Nullable Bundle savedInstan
                     mRvAllergens.setAdapter(mAdapter);
```

Comment:
`mRvAllergens` and `mAdapater` are initialized twice for some reason. Could you help clean up the code here?